sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.controller.service.ts

```diff
@@ -57,6 +60,29 @@ export class ScholasticStandingControllerService {
     };
   }
 
+  /**
+   * Get Scholastic Standing summary details.
+   * @param studentId student id to retrieve the scholastic standing summary.
+   * @returns Scholastic Standing summary details.
+   */
+  async getScholasticStandingSummary(options?: {
+    studentId: number;
+  }): Promise<ScholasticStandingSummaryDetailsAPIOutDTO> {
+    const scholasticStandingSummary =
+      await this.studentScholasticStandingsService.getScholasticStandingSummary(
+        options?.studentId,
+      );
+    const SFASUnsuccessfulCompletionWeeks =
+      await this.sfasIndividualService.getSFASTotalUnsuccessfulCompletionWeeks(
+        options?.studentId,
+      );
+    return {
+      lifetimeUnsuccessfulCompletionWeeks:
+        +(scholasticStandingSummary?.totalUnsuccessfulWeeks ?? 0) +
+        +(SFASUnsuccessfulCompletionWeeks ?? 0),
```

Comment:
Both results are already a number, is there a reason to do the parse again using the "+" at the beginning of these lines?

Reply:
hmm... I was also confused by this. Seeing that they are numbers already, initially, I had not used the `+` operator but then it was concatenating the numbers rather than summing them.

Reply:
If they were concatenated is because they are returned as strings from `getScholasticStandingSummary` and `getSFASTotalUnsuccessfulCompletionWeeks`. Please ensure that, if a `parseInt` is needed, it must happen inside the above-mentioned methods. A method typed to return a number must return a number at runtime.